src/rust/engine/process_executor/src/main.rs

```diff
@@ -40,7 +40,7 @@ If unspecified, local execution will be performed.",
     .get_matches();
 
   let argv: Vec<String> = args
-    .values_of("argv")
```

Comment:
This should be argv (or the "argv" on line 37 needs to change)

The one on line 60 is the one which will cause CI to break